lib/local-links-manager/check_links/link_status_requester.rb

```diff
@@ -6,17 +6,36 @@ class LinkStatusRequester
       delegate :url_helpers, to: "Rails.application.routes"
 
       def call
-        LocalAuthority.all.each do |local_authority|
-          link_checker_api.create_batch(
-            urls_for_local_authority(local_authority),
```

Comment:
We should probably remove this method from line 47, as it's no longer being used.